### ISSUE TITLE
table of contents

### ISSUE THREAD

**Issue by mobularays** [Laverna/laverna#412]
Is there any way to create table of contents in a note? Could we have such a feature?

**Comment by wwwredfish** [comment]
No, currently it isn't possible but we are planning to implement it in one of the future releases.

**Comment by ycopin** [comment]
:+1:
